pkg/virtctl/vm/vm.go

```diff
@@ -194,6 +266,68 @@ func usage(cmd string) string {
 	return usage
 }
 
+func usageAddVolume() string {
+	usage := `  #Dynamically attach a volume to a running VM.
+  {{ProgramName}} addvolume fedora-dv --volume-name=example-dv
+
+  #Dynamically attach a volume to a running VM giving it a serial number to identify the volume inside the guest.
+  {{ProgramName}} addvolume fedora-dv --volume-name=example-dv --serial=1234567890
+
+  #Dynamically attach a volume to a running VM and persisting it in the VM spec. At next VM restart the volume will be attached like any other volume.
+  {{ProgramName}} addvolume fedora-dv --volume-name=example-dv --persist
+  `
+	return usage
+}
+
+func addVolume(vmiName, volumeName, namespace string, virtClient kubecli.KubevirtClient) error {
+	volumeSource, err := getVolumeSourceFromVolume(volumeName, namespace, virtClient)
+	if err != nil {
+		return fmt.Errorf("error adding volume, %v", err)
+	}
+	hotplugRequest := &v1.AddVolumeOptions{
+		Name: volumeName,
+		Disk: &v1.Disk{
+			DiskDevice: v1.DiskDevice{
+				Disk: &v1.DiskTarget{
+					Bus: "scsi",
```

Comment:
Do we want to let the user pick the bus type or is it always scsi the bus type for hotplugged disks?

Reply:
Its always virtio-scsi for hotplugged disks.